python/pyarrow/tests/test_compute.py

```diff
@@ -2069,6 +2098,18 @@ def _check_temporal_rounding(ts, values, unit):
             expected = ts.dt.round(frequency)
             np.testing.assert_array_equal(result, expected)
 
+    if ta.type.tz is None:
```

Comment:
Why is this only done if there is no timezone? (and maybe add a comment about it)

Reply:
Added.